### DISCUSSION TITLE
Welcome to react-checkmark Discussions!

### DISCUSSION THREAD

**Discussion by stanleyxu2005** [stanleyxu2005/react-checkmark#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DmitryTrots** [comment]
Hi! I have a problem with this React component in Firefox browsers - nothing is displayed. In other browsers, the component is displayed fine.
